lib/between_meals/knife.rb

```diff
@@ -73,6 +74,17 @@ def cookbook_upload_all
       exec!("#{@knife} cookbook upload -a -c #{@config}", @logger)
     end
 
+    def berks_cookbook_upload_all
+      @cookbook_dirs.each do |path|
+        cookbooks = Dir['${path}/*'].reject { |o| !File.directory?(o) }
```

Comment:
$path? This isn't shell. :) You want `#{path}` and double quotes. :)

Also, double negatives are sadpanda, `.select { |o| File.directory?(o) }`